cmd/test-utils_test.go

```diff
@@ -77,7 +77,8 @@ func init() {
 	// Set system resources to maximum.
 	setMaxResources()
 
-	logger.Disable = true
```

Comment:
Why was this removed?

Reply:
logger.Disable is not adding value anymore for unit tests as even if we mark it `true` logs were not getting printed. 

This was needed to fix it:
```
80         // Uncomment the following line to see trace logs during unit tests.                                                                                                                             
81         // logger.AddTarget(console.New())                                                                                                                                                               
```